carma_novatel_driver_wrapper/CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+cmake_minimum_required(VERSION 3.5)
+project(carma_novatel_driver_wrapper)
+
+
+find_package(carma_cmake_common REQUIRED)
+# carma_check_ros_version(2)
+carma_package()
+
+find_package(ament_cmake REQUIRED)
+find_package(carma_cmake_common REQUIRED)
+find_package(carma_ros2_utils REQUIRED)
+find_package(carma_msgs REQUIRED)
+find_package(lifecycle_msgs REQUIRED)
+find_package(rclcpp REQUIRED)
+find_package(ros2_lifecycle_manager REQUIRED)
+find_package(sensor_msgs REQUIRED)
+find_package(gps_msgs REQUIRED)
+find_package(carma_driver_msgs REQUIRED)
+find_package(novatel_oem7_msgs REQUIRED)
+
+set(base_dependencies
+  carma_cmake_common
+  carma_ros2_utils
+  carma_msgs
+  rclcpp
+  ros2_lifecycle_manager
+  lifecycle_msgs
+  sensor_msgs
+  gps_msgs
+  carma_driver_msgs
+  novatel_oem7_msgs
+)
+
+set(base_lib_cpp src/carma_novatel_driver_wrapper.cpp)
+carma_define_ament_library(carma_novatel_driver_wrapper "${base_dependencies}" "${base_lib_cpp}")
```

Comment:
I've been testing out ament_auto and I think we should be using that instead. Its a bit more general and less error prone. I think I'll remove these macros. 

```
find_package(ament_cmake_auto REQUIRED)
ament_auto_find_build_dependencies()

include_directories(
  include
)

ament_auto_add_library(<my library> SHARED
  <my cpp file>
)

# Add your rclcpp component call for any components

ament_auto_add_executable(<my exec> SHARED
  <my exec cpp file>
)

target_link_libraries(<my exec> # Link to internal targets
  <my library>
)

# Install
ament_auto_package(
        INSTALL_TO_SHARE config launch
)

```